clients/src/test/java/org/apache/kafka/clients/consumer/MockConsumerTest.java

```diff
@@ -29,7 +29,7 @@
 
     @Test
     public void testSimpleMock() {
-        consumer.subscribe("topic");
+        consumer.subscribe("test");
```

Comment:
Yeah, it was needed because the ConsumerRecords below didn't match the subscribed topic which caused an invalid subscription state.